.circleci/config.yml

```diff
@@ -2,56 +2,11 @@ version: 2.1
 
 jobs:
   build_doc_r:
-    docker:
-      - image: ubuntu:20.04
+    machine:
+      image: ubuntu-2004:202111-01
```

Comment:
To use docker volume mounting, we need to use the machine executor:
https://support.circleci.com/hc/en-us/articles/360007324514-How-can-I-use-Docker-volume-mounting-on-CircleCI-